config/config.json

```diff
@@ -1,9 +1,18 @@
 {
-    "RequiresAuth": false,
+    "RequiresAuth": true,
     "etcdHost": "127.0.0.1",
     "etcdPort": "4001",
-    "hobknobHost": "localhost",
-    "hobknobPort": "3006",
+    "hobknobHost": "127.0.0.1",
+    "hobknobPort": "5432",
+    "AuthProviders":
+        {
+            "AzureAuth":
+            {
+                "AzureClientId": "04fb6794-722d-4b1d-be53-f6d6bc5a62d3",
+                "AzureClientSecret":"OylSOs6WN+r+XSV1+PTToRnipjKefSYsHg/XjRTL0NE=",

```

Comment:
I hope these aren't real secrets! Don't forget this is a public repo. 

Either way, you don't want this change committed